packages/web-components/@sfx/base/src/base.ts

```diff
@@ -0,0 +1,44 @@
+import { LitElement, customElement, html } from 'lit-element';
+import { makeSlot, createChildrenObserver } from './utils';
+
+@customElement('sfx-base')
+export default class Base extends LitElement {
+  observer: any;
```

Comment:
The type of `observer` is known.
```suggestion
  observer: MutationObserver;
```